src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
-package core.basesyntax;
-
-public class Application {
-    public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
-    }
-}
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        Lottery lotteryBall = new Lottery();
+        Ball [] setOfBalls =
+                {lotteryBall.getRandomBall(),
+                 lotteryBall.getRandomBall(),
+                 lotteryBall.getRandomBall()};

```

Comment:
advice on how to improve this:

1. Declare a constant for the amount of balls: `private static final int NUMBER_OF_BALLS = 3`
2. You don't need an array since your goal is just to print information about the balls - therefore you can use a `for` loop and just print `lottery.getRandomBall()` to console